assistive_gym/envs/human_comfort_env.py

```diff
@@ -70,6 +74,86 @@ def _get_obs(self, agent=None): # not needed
     def get_total_force(self):
         pass
 
+    
+    def get_reba_score(self):
+        human_dict = HumanUrdfDict()
+        rebaScore = RebaScore()
+        # list joints in the order required for a reba score
+        joints = ["head", "neck", "left_shoulder", "left_elbow", "left_lowarm", "right_shoulder", "right_elbow", "right_lowarm", 
+            "left_hip", "left_knee", "left_ankle", "right_hip", "right_knee", "right_ankle", "left_hand", "right_hand"]
+        
+        # obtain the links in the right order for the rebascore code
+        dammy_ids = []
+        for joint in joints:
+            dammy_ids.append(human_dict.get_dammy_joint_id(joint))
+
+        # use dammy ids to obtain the right link, use the right knee as the root joint
+        pose = []
+        root = p.getLinkState(self.human.body, dammy_ids[12])[0] # root joint
```

Comment:
why are we using 12 as the index here?

Reply:
For the root joint, the repo explains that it can be any joint
![image](https://github.com/longnguyen2706/assistive-gym/assets/132073581/26046198-6abc-47ca-8e02-049d3a679787)
In their example, they used the right knee joint (or joint 12) as the root so I just followed that exactly (https://github.com/rs9000/ergonomics/blob/78a41799c53b0fb3afbc09d30f0039104e739f80/ergonomics/reba.py#L623)